tests/unit/commands/local/cli_common/test_invoke_context.py

```diff
@@ -44,7 +44,7 @@ def test_must_read_from_necessary_files(self, SamFunctionProviderMock, Container
         )
 
         template_dict = "template_dict"
-        stacks = [Stack("", "", template_file, invoke_context.parameter_overrides, template_dict)]
+        stacks = [Stack("", "", template_file, Mock(), template_dict)]
```

Comment:
`invoke_context.parameter_overrides` is no longer directly passed to `Stack` anymore, and since in this test we don't do anything about parammeter_overrides, I replaced it with `Mock()`